src/gsi/cplr_get_fv3_regional_ensperts.f90

```diff
@@ -1,14 +1,21 @@
 module get_fv3_regional_ensperts_mod
 use abstract_get_fv3_regional_ensperts_mod,only: abstract_get_fv3_regional_ensperts_class
   use kinds, only : i_kind
+  use general_sub2grid_mod, only: sub2grid_info
+  use constants, only:max_varname_length
   type, extends(abstract_get_fv3_regional_ensperts_class) :: get_fv3_regional_ensperts_class
   contains
     procedure, pass(this) :: get_fv3_regional_ensperts => get_fv3_regional_ensperts_run
     procedure, pass(this) :: ens_spread_dualres_regional => ens_spread_dualres_regional_fv3_regional
     procedure, pass(this) :: general_read_fv3_regional
-    procedure, pass(this) :: general_read_fv3_regional_dirZDA
   end type get_fv3_regional_ensperts_class
+    type(sub2grid_info):: grd_fv3lam_ens_dynvar_io_nouv,grd_fv3lam_ens_tracer_io_nouv,grd_fv3lam_ens_uv
+    character(len=max_varname_length),allocatable,dimension(:) :: fv3lam_ens_io_dynmetvars3d_nouv  ! copy of cvars3d excluding uv 3-d fields   CV
+    character(len=max_varname_length),allocatable,dimension(:) :: fv3lam_ens_io_tracermetvars3d_nouv  ! copy of cvars3d excluding uv 3-d fields   CV
```

Comment:
These lines exceed the 132 character limit.  Please reduce the number of characters on these lines.